src/classes/VOL_SharedCode_TEST.cls

```diff
@@ -99,6 +99,69 @@ public with sharing class VOL_SharedCode_TEST {
 			system.assertEquals(1, listCon2.size());
 		}
     }
+<<<<<<< HEAD
```

Comment:
Christian, it looks like you got some diff/merge tags accidentally interjected into the source file.  can you clean up this file and resubmit it.  thanks.